src/portfolios/portfolio/components/TaskOrder/TaskOrderDetails.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <div class="_task-order-details">
+    <div class="d-flex align-center">
+      <a
+      @click="handleClick"
```

Comment:
- add an id
- add the keyboard events
- add `role="button"` and `tabindex="0"`